flattentool/input.py

```diff
@@ -472,21 +484,66 @@ def read_sheets(self):
             except ValueError:
                 pass
         self.sub_sheet_names = sheet_names
+        self.sheet_names_map = OrderedDict((sheet_name, sheet_name) for sheet_name in sheet_names)
```

Comment:
I'm not sure we need this for the CSV implementation? I can't see where else it's used besides the XLSX implementation....

Reply:
Just spoke with @kindly, this is needed in order to add CSV metatab support in future, so useful to leave it in.